### ISSUE TITLE
SetFile Service

### ISSUE THREAD

**Issue by kevinherron** [beanit/iec61850bean#13]
Are there plans to implement ACSI SetFile / MMS ObtainFile request?

**Comment by sfeuerhahn** [comment]
no, I don't have any plans currently. I'm happy to take contributions though.
